src/video/video.js

```diff
@@ -136,7 +136,7 @@
             // sanitize potential given parameters
             settings.doubleBuffering = !!(settings.doubleBuffering);
             settings.autoScale = (settings.scale === "auto") || false;
-            settings.maintainAspectRatio = !!(settings.maintainAspectRatio);
+            settings.scaleMethod = !!(settings.scaleMethod);
```

Comment:
It'll work, but you're already doing an if-then-else check later in the code, and never *explicitly* checking for "default" anyway.

You can also use a little trick with an array:

```javascript
settings.scaleMethod = [ "fill", "stretch" ].indexOf(settings.scaleMethod) >= 0 ? settings.scaleMethod : "default";
```